fastai2/data/external.py

```diff
@@ -243,4 +249,6 @@ def untar_data(url, fname=None, dest=None, c_key='data', force_download=False, e
         if _get_check(url) and _check_file(fname) != _get_check(url):
             print(f"File downloaded is broken. Remove {fname} and try again.")
         extract_func(fname, dest.parent)
+        print(f'fname: {fname}, dest: {dest}')
```

Comment:
We try to avoid being verbose. Maybe just make a test to see if a rename is necessary and just print something then?

Reply:
Oh I forgot to remove this...

Reply:
I refactored the code, to check outside, I added a test. It is complicated to add an integral test without a missnamed downloadable dataset.
It is so painful that the unzip and untar methods don't give back the extrated folder.